hack/verifyGuides.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Why keep this in a new hack folder vs inside scripts?

Reply:
Good point. I will move the file